jsf-showcase-webapp/src/test/java/com/liferay/faces/test/showcase/miscellaneous/MiscellaneousTester.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2000-2016 Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.liferay.faces.test.showcase.miscellaneous;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import org.junit.Assert;
+
+import com.liferay.faces.test.selenium.Browser;
+import com.liferay.faces.test.selenium.assertion.SeleniumAssert;
+import com.liferay.faces.test.showcase.TesterBase;
+
+
+/**
+ * @author  Kyle Stiemann
+ * @author  Philip White
+ */
+public class MiscellaneousTester extends TesterBase {
+
+	protected void assertElementContentEqualsFileContent(Browser browser, String fileContent) {
```

Comment:
This method still needs more work. Please make sure that the method signature is `assertElementContentEqualsFileContent(Browser browser, String elementXpath, String filePath)` (as I said in [this comment](https://github.com/stiemannkj1/liferay-faces-showcase/pull/9#discussion_r86589976)). Changing the method to take the `elementXpath` and the `filePath` will allow the method to be more generic and usable for the Alloy Showcase, and it will reduce some duplicate code.